docs/pipeline/rules/headerscollection.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: "Headers Collection"
+description: "The \"Headers Collection\" rule can be used to add span attributes by collecting HTTP Headers data."
+sidebarTitle: "Headers Collection"
+icon: "webhook"
+---
+
+<Info>
+  This Instrumentation Rule is currently only available with the Odigos **Enterprise** plan.<br />
+  [Contact us](https://odigos.io/) for more information.
+</Info>
+
+## Considerations
+
+<Warning>
+  Before enabling **headers collection**, please note the following:
+  - The support for headers collection varies between instrumentation libraries and languages.
+</Warning>
+
+## Configuration Options
+
+<AccordionGroup>
+  <Accordion title="headerKeys">
+    **headerKeys** `string[]` : Limit payload collection to specific header keys.
```

Comment:
I think it can be valuable to also describe how this key is collected.

e.g. that an header key "foo" with value "bar" will be collected as span attribute named `http.request.header.foo` with attribute value `["bar"]` (or `http.response.header.foo` if this is for response)

@edeNFed just making sure java is implemented this way ([semconv](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/registry/attributes/http.md#http-request-header))